### PR TITLE
Purchase vat  

### PR DESCRIPTION
**Pull request by emran-wedevs**

### CODE REVIEW

File: includes/updates/update-1.6.9.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    }
+
+
+}
+
+
+function crate_erp_acct_purchase_details_tax_table_1_6_9() {
+
+    global $wpdb;
+    $sql = "CREATE TABLE IF NOT EXISTS {$wpdb->prefix}erp_acct_purchase_details_tax (
```
Comment (tanmayjay) [inline includes/updates/update-1.6.9.php:55]: 'IF NOT EXISTS' is not needed for table creation while using dbDelta

File: includes/updates/update-1.6.9.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              `id` int(20) UNSIGNED NOT NULL AUTO_INCREMENT,
```
Comment (tanmayjay) [inline includes/updates/update-1.6.9.php:56]: No need to use backticks(` `) for column name

File: includes/updates/update-1.6.9.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+              `invoice_details_id` int(20) NOT NULL,
+              `agency_id` int(20) DEFAULT NULL,
+              `tax_rate` decimal(20,2) NOT NULL,
+              `created_at` timestamp DEFAULT NULL,
+              `created_by` int(20) DEFAULT NULL,
+              `updated_at` timestamp DEFAULT NULL,
+              `updated_by` int(20) DEFAULT NULL,
+              PRIMARY KEY (`id`)
+            ) DEFAULT CHARSET=utf8";
```
Comment (tanmayjay) [inline includes/updates/update-1.6.9.php:65]: You may use $wpdb->get_carset_collate for charset

File: includes/updates/update-1.6.9.php [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    $sql = "CREATE TABLE IF NOT EXISTS {$wpdb->prefix}erp_acct_purchase_details_tax (
+              `id` int(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+              `invoice_details_id` int(20) NOT NULL,
+              `agency_id` int(20) DEFAULT NULL,
+              `tax_rate` decimal(20,2) NOT NULL,
+              `created_at` timestamp DEFAULT NULL,
+              `created_by` int(20) DEFAULT NULL,
+              `updated_at` timestamp DEFAULT NULL,
+              `updated_by` int(20) DEFAULT NULL,
+              PRIMARY KEY (`id`)
```
Comment (tanmayjay) [inline includes/updates/update-1.6.9.php:64]: There should be two spaces after 'PRIMARY KEY'. i.e. `PRIMARY KEY  (`id`)`

File: includes/updates/update-1.6.9.php (hunk 1 above)
Comment (emran-wedevs) [reply includes/updates/update-1.6.9.php:55]: We decided to follow the previous flow for sql

File: includes/updates/update-1.6.9.php (hunk 2 above)
Comment (emran-wedevs) [reply includes/updates/update-1.6.9.php:56]: We decided to follow the previous flow for sql

**Review by tanmayjay — APPROVED** [review]
Considering the author's response on previously suggested amendments, everything looks okay.

**Comment by Rink9** [comment]
@emran-wedevs 
**Issue** 
tax/VAT should appear while making transactions.

**Pull request closed without merging**